prisma/schema.prisma

```diff
@@ -8,23 +8,26 @@ datasource db {
 }
 
 model Comment {
-  id          Int       @id @default(autoincrement())
-  accidentId  String    @db.VarChar
-  text        String?
-  authorId    String    @db.VarChar
-  isPublished Boolean   @default(false)
-  createDate  DateTime? @default(now()) @db.Date
-  updateDate  DateTime? @default(now()) @db.Date
-  author      User      @relation(fields: [authorId], references: [id])
+  id          Int      @id @default(autoincrement())
+  accidentId  String   @map("accident_id") @db.VarChar
+  text        String
+  authorId    String   @map("author_id") @db.VarChar
+  isPublished Boolean  @default(false) @map("is_published")
+  createdAt   DateTime @default(now()) @map("created_at")
+  updatedAt   DateTime @default(now()) @map("updated_at")
```

Comment:
I noticed that `@db.Date` was not giving us time, so I removed this annotation. The field is now `timestamp without timezone`. I also switched from `createDate` / `updateDate` to `createdAt` / `updatedAt` as this is an industry standard.